src/main/java/nl/smallproject/www/techiteasy/dtos/CiModule/CiModuleInputDto.java

```diff
@@ -0,0 +1,44 @@
+package nl.smallproject.www.techiteasy.dtos.CiModule;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.PositiveOrZero;
+import jakarta.validation.constraints.Size;
+
+public class CiModuleInputDto {
+
+    @NotBlank(message = "{cimodule.name.notblank}")
```

Comment:
Leuke oplossing heb je hier bedacht, maar ik betwijfel of het op deze manier echt efficienter is. 
Misschien dat het wel handig is als je messages kunt hergebruiken. Nu heb je voor elke annotatie voor elke entity een andere message gemaakt en had je het eigenlijk net zo goed hier kunnen uitschrijven als string.